src/content/docs/en/guides/view-transitions.mdx

```diff
@@ -235,6 +235,34 @@ You can use this event to run code on every page navigation, or only once ever:
   }, { once: true });
 </script>
 ```
+
```

Comment:
Given that we now have `astro:load` *and* `astro:beforeload`, we probably do need a line or two about the difference, rather than just relying on people reading the description of each and working it out.

I’d suggest an `h3` before `astro:load`, e.g. `### Useful events` or something with one or two sentences describing the difference.

May also be good to update the first sentence under `astro:load`? That says “An event that fires on page navigation” but how is that different from the `beforeload` “after the new page replaces the old page”?

Reply:
Hm, is that common? It seems like it's just going to be a repeat of the sentences in each of the event descriptions. 

> May also be good to update the first sentence under astro:load?

yes, this one can be more specific.

Reply:
Updated the astro:load.